### PR TITLE
Add Invictus Capital

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: _data/betting.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ websites:
     tfa:
       - totp
     doc: https://www.cloudbet.com/en/2_step_authentication
-
+
```
Comment (Carlgo11) [inline _data/betting.yml]:
```suggestion

```

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -399,7 +399,15 @@ websites:
     tfa:
       - totp
     doc: https://support.hitbtc.com/en/support/solutions/articles/63000224976
-
+
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion

```

File: _data/cryptocurrencies.yml [hunk 3]
```diff
@@ -399,7 +399,15 @@ websites:
...
-
+    
+  - name: Invictuscapital
+    url: https://invictuscapital.com
+    img: invictuscapital.png
+    tfa:
+      - sms
+      - phone
+      - totp
+
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request.

I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

File: _data/cryptocurrencies.yml (hunk 2 above)
Comment (40504) [reply _data/cryptocurrencies.yml]: done ✅

File: _data/cryptocurrencies.yml (hunk 3 above)
Comment (40504) [reply _data/cryptocurrencies.yml]: done ✅

File: _data/cryptocurrencies.yml [hunk 4]
```diff
@@ -400,6 +400,14 @@ websites:
       - totp
     doc: https://support.hitbtc.com/en/support/solutions/articles/63000224976
 
+  - name: Invictuscapital
```
Comment (phallobst) [inline _data/cryptocurrencies.yml]:
```suggestion
  - name: Invictus Capital
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by 40504** [comment]
> Thanks for the pull request.
> 
> I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?
> 
> If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
> **Remember to blur or crop out any personal information linked to you.**
> 
> You can attach the screenshots here in a comment, like this:
> screenshot

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**